server/job_service.go

```diff
@@ -302,6 +303,9 @@ func (s *jobService) RunJob(jobType job.Type, callID, postID, jobID, authToken s
 		transcriberConfig.PostID = postID
 		transcriberConfig.TranscriptionID = jobID
 		transcriberConfig.AuthToken = authToken
+		if val := os.Getenv("MM_CALLS_TRANSCRIBER_NUM_THREADS"); val != "" {
+			transcriberConfig.NumThreads, _ = strconv.Atoi(val)
```

Comment:
Should we be handling the error with a default, or assume it's correct (because its an admin-controlled env variable)? (Just curious about that kind of thing system-wide, @jupenur )

Reply:
`calls-transcriber` implements validation and errors on a zero value. In general we should strive to validate all inputs regardless of the level of trust assumed just to avoid mistakes resulting from bad assumptions, and to make a habit of it. That said, there'd be no direct security impact here even if there was no validation.

0/5 on whether we'd want to add some extra error handling here, too, just to make the admin UX better.

Reply:
Yes, this is really there to help me test and/or for very advanced use cases at the moment. Until we surface this through proper config I am okay with keeping it this way as the ultimate validation is done on the other side and when zero (error case) a default is used.